### DISCUSSION TITLE
Will Apple Hypervisor support amd64?

### DISCUSSION THREAD

**Discussion by BlackHole1** [containers/podman#18587] [Q&A] [answered]
I have reviewed the code and found that currently `applehv` can only be built on `arm64` architecture, but not on `amd64` (Intel chip) architecture. Does this mean that podman's `applehv` only supports `arm64` (m1/m2)?  🤔
If so, may I inquire about the reason behind this decision 🙇‍♂️? Because I noticed that the `Apple Hypervisor` does support the amd64 (Intel chip) architecture.

/cc @baude

**Comment by baude** [comment]
links on support please?

**Reply by BlackHole1** [reply]
Sorry. I forget about it.

https://github.com/containers/podman/blob/f4185396427e2f76bc12e2a4394d182f56fddd0d/cmd/podman/machine/platform_darwin.go#L1

**Comment by baude** [comment]
@BlackHole1 My apologies ... I meant do you have a link that shows Intel Apple hardware supports the Apple Hypervisor?

**Reply by BlackHole1** [reply]
Apple: https://developer.apple.com/documentation/hypervisor/intel-based_mac

BTW. I have tested by installing UTM and found that Intel can support Apple Hypervisor. Moreover, UTM is completely open source. https://github.com/utmapp/UTM

**Comment by baude** [comment, marked as answer]
Ok, looks like it does work on Intel Mac's.  I believed it did not.  In that case, we would enable it for Intel.  I am working on the underpinning of this right now and when it is done, I can circle back and make this the case.

**Reply by BlackHole1** [reply]
Thanks!

**Reply by VacheK19** [reply]
thx)

**Comment by BlackHole1** [comment]
Hi @baude . Is there any progress at the moment? I would like to know at which stage the support for applehv currently stands. Is there a planned completion time? Can I use applehv with the latest code now? Would there be any issues with that? Thanks🙇‍♂️.
